src/manifolds/PositiveNumbers.jl

```diff
@@ -0,0 +1,224 @@
+@doc raw"""
+    PositiveNumbers <: Manifold{ℝ}
+
+The hyperbolic manifold of positive numbers $H^1$ is a the hyperbolic manifold either
+represented by just positive numbers or the [Poiincaré half-plane model](https://en.wikipedia.org/wiki/Poincaré_half-plane_model).
```

Comment:
```suggestion
represented by just positive numbers or the [Poincaré half-plane model](https://en.wikipedia.org/wiki/Poincaré_half-plane_model).
```

Reply:
oh sorry, this was left over, I first though on C one could do the half-plane model, but that would be a bad name to do that here. So I have to delete that line anyways.